src/twilio/functions/sms/intake-delivery-txts.js

```diff
@@ -20,7 +20,6 @@ exports.handler = function codeSms(context, event, callback) { // eslint-disable
   // below: looks up code in request Airtable, gets all important values about the requesting neighbor
   base("Requests")
     .select({
-      view: "Grid view", // from staging base
```

Comment:
@mab253 oh sorry my Q was this change intentional? just checking since this wasnt mentioned in the desc!

Reply:
o yes! i believe this is already deployed via the twilio flow - but this is because in the "Requests" view that's not the Staging base, there's no view called "Grid view." And if we don't add the view parameter, it searches through the whole table 👍 

thnx for flagging!!